PI-Vision-Extensibility/README.md

```diff
@@ -0,0 +1,21 @@
+# PI Vision Extensibility
+This repository contains sample HTML/CSS/JavaScript for creating custom symbols for PI Vision.
+
+## Overview
+The sample code contained in this repository provides code to extend PI Vision with new custom symbols.  
+
+### Feedback
+
+To request a new sample, if there is a feature or capability you would like demonstrated, or if there is an existing sample you would like in your favorite language, please give us feedback at the [OSIsoft GitHub Feedback Channel](https://feedback.osisoft.com/forums/922279-osisoft-github). To post feedback, submit feature ideas, or report bugs, use the [OSIsoft GitHub Feedback Channel](https://feedback.osisoft.com/forums/922279-osisoft-github). 
+
+### Support
+
+If your support question or issue is related to an OSIsoft product (error message, problem with product configuration, etc.), please open a case with OSIsoft Tech Support through the [OSIsoft Customer Portal](https://my.osisoft.com/).
+
+If your support question or issue is related to a non-modified sample (or test) or sample documentation; please email [Samples@osisoft.com](mailto:Samples@osisoft.com). 
+
+### License
+
+The PI Vision Extensibility Samples in this repository are licensed under the [Apache 2 license](https://github.com/osisoft/OSI-Samples/blob/master/LICENSE.md).
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/PI%20Vision/extensibility-sample.unit-tests?branchName=47718_ExtensibilitySample)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=1311&branchName=47718_ExtensibilitySample)
```

Comment:
This build status doesn't make sense I would either get rid of it for now, or setup a table for when there are more symbols

Reply:
I'll remove the Build Status section for now.